docker-compose.production.yml

```diff
@@ -44,17 +49,26 @@ services:
       - webnet
     secrets:
       - petabox_seed
-
   infobase:
     restart: always
-    hostname: ${HOSTNAME:-$HOST}
+    deploy:
+      replicas: 1
+    hostname: "$HOSTNAME"
     environment:
       - INFOBASE_OPTS= fastcgi
       - INFOBASE_CONFIG=/olsystem/etc/infobase.yml
       - PYENV_VERSION=3.8.6
     volumes:
       - ../olsystem:/olsystem
       -  /1:/1
+  infobase_nginx:
+    image: nginx:1.4.6
+    restart: always
+    deploy:
+      replicas: 1
+    expose: 7000
```

Comment:
Ports will have do something similar to how ports/expose are handled in the covers PR.